backend/src/schemas/user.py

```diff
@@ -20,6 +20,7 @@ class UserOut(BaseModel):
     auth_provider: str
     created_at: datetime
     updated_at: datetime
+    is_guest: bool
```

Comment:
Do we need this so often it needs a bool instead of `user.role == 'guest'`? Feels a bit like double-info, but not against if it simplifies stuff.

Reply:
is_guest is overkill yes, but a nice QoL imo